packages/manager/modules/iam/src/components/actionSelect/actionSelect.controller.js

```diff
@@ -388,4 +425,20 @@ export default class ActionSelectController {
   static countSelectedActions(actions) {
     return actions?.filter(({ selected }) => selected).length;
   }
+
+  /**
+   * count the number of actions matching searchQuery
+   * @param {Action[]} actions
+   * @param {string} searchQuery
+   * @returns {Number}
+   */
+  static countActionsMatchingSearch(actions, searchQuery) {
+    if (!actions || !searchQuery) {
+      return 0;
+    }
+    return actions.filter(
+      (action) =>
+        action?.value.toLowerCase().indexOf(searchQuery.toLowerCase()) > -1,
```

Comment:
```suggestion
        action.value.toLowerCase().indexOf(searchQueryLowerCase) > -1,
```
Create a constant `const searchQueryLowerCase = searchQuery.toLowerCase()` to avoid repeating the operation in the loop

Reply:
Done